spring-boot/074771ec125dd407af0282b92960e9e9e3377e84/original/README.md

```diff
@@ -26,7 +26,7 @@
     </plugin>
 ```
 
-## 5. Inside the folder  **spring-boot** run the command:
+## 5. Inside the folder **spring-boot/spring-boot** run the command:

```

Comment:
Neste caso, acontece o mesmo problema envolvendo dependências que não podem ser resolvidas por conta da ausência da versão.
`[ERROR]     'dependencies.dependency.version' for org.springframework:spring-context:jar is missing.`

Reply:
Resolvido, foi necessário modificar a versão na mão. A atual não estava mais no repositório.

Reply:
Resolvido.